CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # Changelog
 
+## 0.1.24
+
+### Breaking changes
+
+- `uv pip uninstall` no longer supports editable requirements ([#2577](https://github.com/astral-sh/uv/pull/2577))
```

Comment:
I think this should be "the `-e` flag" or something. You can still uninstall editable installs, you just can't provide editable requirements. (I know it says "editable requirements", I just think it will be misinterpreted as more than it is.)

Reply:
Done. Can you clarify that in the pull request body?